### ISSUE TITLE
Non 48 hour server/Non Advanced Server experiences Apocalypse/Possibly No Chimes Heard By Anyone Interested in Stopping an Apocalypse

### ISSUE THREAD

**Issue by DougLefelhocz** [jasonrohrer/OneLife#1017]
Kortyna posted this image [on the discord](https://cdn.discordapp.com/attachments/328215300279500800/1190793699840491660/image.png?ex=65a3180d&is=6590a30d&hm=d751762c2e71836a7a4df563020182ecbe800f58d586dec3907e0734d391cb15&) from a server1 life.

[Jason Rohrer](https://onehouronelife.com/forums/viewtopic.php?id=4877):

"What if, _at the top of the tech tree_ [emphasis added], the most difficult-to-craft item was The Button?"

Of course, "at the top of the tech tree" "the button" didn't work out the first time.  But, families getting or not getting to the top of the tech tree I still think relevant.

To my knowledge, there was no glass, no radios, no trucks, no oil wells, and no paved roads on server1.  Also, no cameras, I'm guessing.  No advanced towns also.  I did an oil pumpjack on server1 during off hours/only bs2 had incoming players not checking a custom server and someone else who joined me made some more kerosene.  But, I did NOT use it, nor wander back to that town and I believe that doing another oil pumpjack would have made more sense than wandering back to that kerosene, since I did that two or three days ago now.

I also mentioned elsewhere that I had to take down an apocalypse tower earlier in the week.  

Earlier today I had made the argument that Gingers had a valid reason to resettle, since no server1 family has exhausted the game's content this past week, and they could potentially survive in an older town if they got oil up.

About 30 minutes before the apocalypse happened I had visited blacks and tans as [Pippin Harvest](http://lineage.onehouronelife.com/server.php?action=character_page&id=10053786) trying to get rubber supplies, and one had a shallow well, the other just had a ready well site.  It's not like s1 was just advanced towns.

The person who says that s/he did the apocalypse is the same person who expressed anger on the discord at me having taken down the tower this week.  If they were smart about their attempt, they made the key steps when incoming players not checking a custom server were all directed to bigserver2.  I even asked this person "did anyone even hear the two chimes?"  He/she had no response.

"Rare, unsustainable ingredients that you cannot procure while working completely alone."

No, someone can just birth their alt.  In fact, I saw two similar PX names for a bit a few days ago on s1, similar to the person's discord name.  I find it likely that this person did all by their own self. 

"World-wide warnings as the ritual gets closer and closer to completion."

I played as Spoorthi Harvest, Love Harvest, and I think 2 or 3 other lives today in [Harvest family](http://lineage.onehouronelife.com/server.php?action=character_page&id=10052506).  I also played on s1 in [Appler family](http://lineage.onehouronelife.com/server.php?action=character_page&id=10048714). and a few lives in [Hohenwarter](http://lineage.onehouronelife.com/server.php?action=character_page&id=10047358) family before that.  I did not hear a chime in any of those lives, and haven't heard a chime after I took down the previous tower.  Again, I wonder, did anyone **at all** hear the chimes?  Or a second chime?  If no one hears the chime, I don't see how there's any sort of drama.

"Toward the end in a steady state, life is meaningless, and no contribution matters."

I agree on this, and hence generally have either played or tried to help newer families.  But, there is something to getting oil up (AND then improving the process by doing an oil well), doing a paved road, glass, and so on.  There's also something to advanced towns when I played on bs2 a few lives.  I do not agree that this "steady state" had gotten achieved for a single family on server1 when this happened.  Nor the server as a whole.  Again, I observed two early camps.

So, I think we have a non-intended use of the apocalypse here.

Also, as I write this, the arc on bigserver2 has been going on for 917851 OHOL years, some 637 days old.  Before the server merge especially, a very strong case, could get made that an apocalypse should have happened a long time ago, or an arc reset was long overdue.  But, no apocalypse happened (for a variety of reasons).

And instead there's an apocalypse on a server where there's plenty of content that hadn't become relevant, where not a single person might have taken down the tower, other the person who built it.

Edit: While I wrote the above, apparently Bluehawk said this:

"BlueHawk — Today at 7:13 PM
Not bugged. I waited til no one was on to hear the chime"

**Comment by DougLefelhocz** [comment]
Maybe a server needs to have at least 40 players on it for a week... or a month... at all times... before an apocalypse is possible.  Or just no apocalypses at all, since they don't seem needed even when there's a case for their use.  Or apocalypses are only possible on one server.

**Comment by DougLefelhocz** [comment]
Later:

"BlueHawk — Today at 8:03 PM

I actually agree.  I feel like you shouldn't be able to put the blocks on the tower if the server is below 15 or something, cus it only dings when you go up to the block thats seeping.  So by only being able to put it on while theres 15 or ppl then that means that at least 15 ppl hear the ding."

**Comment by cordy-123** [comment]
**TL;DR**  
 
A player is doing all 4 levels of the endtower **ALONE** by himself in server1 and triggering it when the population is above 15 players (when players are sent to server1 during server splits and during game updates).

Since he is making the endtower when no one is in server1, no one can hear the _monumentcall_. 
 
Therefore, apocalypse on server1 is basically unstoppable.

-----------

_**Recommendations:**_

Change **ALL** levels of the endtower so that they can only be made when there is at least **15 players** in the server.
(The apocalypse itself ALREADY needs 15 players to trigger. https://github.com/jasonrohrer/OneLife/commit/08774f687bd9b9218c78f8b80405672cea2e01a0)
  
Change **ALL levels** of the endtower **AND** the **apocalypse itself** to have the tag **"_+nearPop_50_10000_"**.

**Comment by cordy-123** [comment]
This is a problem that hits server1 particularly.   
     
The problem is that: 

1. you **CAN** put up the endblocks into the endtower **EVEN if there are LESS than 15 people in the server**.

(However, you can trigger the actual apocalypse using the bloody knife **ONLY** with 15 people in the server. https://github.com/jasonrohrer/OneLife/commit/08774f687bd9b9218c78f8b80405672cea2e01a0)

2. when Mr. Rohrer changed it so that you have to be close to most of the server population to make an endtower https://github.com/jasonrohrer/OneLifeData7/commit/1b2ff8ddbd7b799204adf9bc7e23227495a9a4ac in response to Bluehawk and others moving hundreds of thousands of tiles east https://github.com/jasonrohrer/OneLife/issues/905, Mr. Rohrer only changed it for the **FIRST** level of the endtower.


1st level of the endtower - has the tag "_+nearPop_50_10000_" -> https://onetech.info/2477-Endtower-Base-nearPop_50_10000 
 
The other layers of the endtower **AND** the actual apoc itself -> **DO NOT** have the tag "_+nearPop_50_10000_"

-----------

_This is how they do it._

They set up the endtower (out of bands so no one can accidentally stumble upon it) all the way to level 4 ALONE in the server **so no one actually hears the monumentCall**.

And then when the population goes up, they go to their tower and trigger the apoc. 

This is done during game updates (when people are sent off to s1) or during server splits. 

The timing of both are VERY EASY to figure out.

(Server splits happen when bs2 reaches a population of 120. 

Game update server restarts usually happen after a #live-dev-changes notification and you can see that bs2 is shutting down by looking at the server status webpage.)

-----------

This does not feel very "sporting" as the whole idea of the apoc is a battle between factions of the server to serve as a story / experience.

Having one person setup all 4 levels of the apoc in secret _alone_ by himself when no one else is in the server and then, the moment the population goes up, run to their tower (with their alt as a sacrifice) to finish up the apoc is not really in-line with that.

 
-----

**Comment by DougLefelhocz** [comment]
Bluehawk on January 4, 2024 in game-pictures:

"Did the same thing  before too. ... a couple months ago during an update I apoc'd it."

**Comment by DougLefelhocz** [comment]
Forman [reported](http://onehouronelife.com/forums/viewtopic.php?id=10842) this also.

**Comment by jasonrohrer** [comment]
Ah, nice find, folks!  Will fix.

**Comment by jasonrohrer** [comment]
Oh, I see the problem.. the nearpop_50 thing is saying, "50% of the population must be near you."

But if the server is empty...
